tests/good-api-test.js

```diff
@@ -258,6 +258,31 @@ describe("Translations API", function () {
     }).not.throw();
   });
 
+  it("The uploadTranslationInstanceMethod function puts translation content", function (done) {
+    should(function(){
+      nock("https://www.transifex.com")
+        .put("/api/2/project/transifex/resource/sample/translation/es/?file")
+        .reply(201, {
```

Comment:
@alicoding I spend a bunch of time looking at this and determined that this test passes but there's some side effect that makes further tests fail due to interfering with `nock`

any advice? i'm stuuuuck